server/util-server.js

```diff
@@ -66,17 +66,18 @@ exports.decodeJwt = (jwt) => {
 };
 
 /**
- * Gets a Access Token form a oidc/oauth2 provider
- * @param {string} tokenEndpoint The token URI form the auth service provider
+ * Gets an Access Token from an oidc/oauth2 provider
+ * @param {string} tokenEndpoint The token URI from the auth service provider
  * @param {string} clientId The oidc/oauth application client id
  * @param {string} clientSecret The oidc/oauth application client secret
- * @param {string} scope The scope the for which the token should be issued for
- * @param {string} authMethod The method on how to sent the credentials. Default client_secret_basic
+ * @param {string} scope The scope(s) for which the token should be issued for
+ * @param {string} authMethod The method used to send the credentials. Default client_secret_basic
  * @returns {Promise<oidc.TokenSet>} TokenSet promise if the token request was successful
  */
-exports.getOidcTokenClientCredentials = async (tokenEndpoint, clientId, clientSecret, scope, authMethod = "client_secret_basic") => {
+exports.getOidcTokenClientCredentials = async (tokenEndpoint, clientId, clientSecret, scope, audience, authMethod = "client_secret_basic") => {
     const oauthProvider = new oidc.Issuer({ token_endpoint: tokenEndpoint });
     let client = new oauthProvider.Client({
+        issuer: audience,
```

Comment:
can this change the behaviour if the issuer audience was prviously not provided? is the issuer and the audience the same concept?

Reply:
Thanks for catching this. I forgot to delete it when I was trying to investigate the Client in my IDE 😬

This should be passed correctly however in the `AuthorizationParameters` in `grantParams` below this:
https://github.com/panva/openid-client/blob/v5.x/types/index.d.ts#L100

I set the knex migration to make `oauth_audience`'s default `null`, which I would think should work the same as `scope` does now